package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -452,8 +518,10 @@ def _update_water_selection(self):
                           ' {} atoms'.format(len(self._water)))
         if self.filter_first:
             ns_water_selection = AtomNeighborSearch(self._water)
-            self._water = ns_water_selection.search(self._s1, 3. * self.distance)
-            self._water = ns_water_selection.search(self._s2, 3. * self.distance)
+            self._water = ns_water_selection.search(self._s1, 3. *
```

Comment:
I'd break the 79 char rule and leave it one one line. If doing a line break, indent properly and break earlier so that each line contains "complete" units of meaning, e.g.,
```python
self._water = ns_water_selection.search(self._s2, 
                                        3. * self.distance)
```
Your editor should be indenting towards opening parentheses...

Reply:
Please indent for legibility. Follow [PEP8 indentation rules](https://www.python.org/dev/peps/pep-0008/#indentation)